Library/Homebrew/cmd/formulae.sh

```diff
@@ -3,6 +3,21 @@
 #:  List all locally installable formulae including short names.
 #:
 
+smart-sed() {
```

Comment:
This seems like a lot of code for a single flag replacement per OS. I've opened https://github.com/Homebrew/brew/pull/9389 for a simpler approach